libr/core/fortune.c

```diff
@@ -1,58 +1,112 @@
-/* radare2 - LGPL - Copyright 2009-2020 - pancake */
+/* radare2 - LGPL - Copyright 2009-2022 - pancake, condret */
 
 #include <r_core.h>
-
-static const char *fortunes[] = {
-	"tips", "fun",
-};
+#include <r_util.h>
 
 static char *getFortuneFile(RCore *core, const char *type) {
+	char *home = r_sys_getenv (R_SYS_HOME);
+	if (!home) {
+		return NULL;
+	}
+	char *path = r_str_newf (R_JOIN_3_PATHS ("%s", R2_HOME_FORTUNES, "fortunes.%s"),
```

Comment:
The R_JOIN_N_PATHS are imho an awful collection of defines that should not exist. its preferible to use `r_file_new()` or R_SYS_DIR manually

Reply:
uhm, I'd like to do this later, bc rn r_join_n_paths is used almost everywhere. I think we need to come up with something consisten for this